### PR TITLE
RUMM-560 Update integration tests for view loading time

### PR DESCRIPTION
**Pull request by mariusc83**
### What does this PR do?

A brief description of the change being made with this pull request.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: tools/unit/src/main/kotlin/com/datadog/tools/unit/assertj/JsonObjectAssert.kt [hunk 1]
```diff
@@ -433,20 +433,49 @@ class JsonObjectAssert(actual: JsonObject) :
...
-                is BigInteger -> hasField(key, value)
-            }
+            checkHasField(value, key)
         }
         return this
     }
 
+    private fun JsonObjectAssert.checkHasField(
+        value: Any?,
+        key: String
```
Comment (ncreated) [inline tools/unit/src/main/kotlin/com/datadog/tools/unit/assertj/JsonObjectAssert.kt:443]: It's ok 👌 , but a bit weird as: `key`, `value` feels more natural than reversed: `value`, `key`.

File: instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/rum/EndToEndRumFragmentTrackingTests.kt [hunk 2]
```diff
@@ -39,84 +39,131 @@ internal class EndToEndRumFragmentTrackingTests {
...
+        // view stopped
         expectedEvents.add(
             ExpectedViewEvent(
-                fragmentCViewUrl,
+                fragmentAViewUrl,
                 3,
-                currentFragmentExtras()
+                currentFragmentExtras(),
+                extraViewAttributes = mapOf(
+                    "loading_type" to "fragment_redisplay"
```
Comment (buranmert) [inline instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/rum/EndToEndRumFragmentTrackingTests.kt:151]: how does that work?

Comment (mariusc83) [reply instrumented/integration/src/androidTest/kotlin/com/datadog/android/sdk/integration/rum/EndToEndRumFragmentTrackingTests.kt:151]: We are instrumenting the test application to perform swipes between the fragments. We are intercepting the events as you are doing with your Python mock server and we are asserting them at the end. In this case this fragment (view) was already displayed once so when we detect that and send `fragment_redisplay` for the `loading_type`

File: tools/unit/src/main/kotlin/com/datadog/tools/unit/assertj/JsonObjectAssert.kt (hunk 1 above)
Comment (mariusc83) [reply tools/unit/src/main/kotlin/com/datadog/tools/unit/assertj/JsonObjectAssert.kt:443]: Oh...that was not needed anymore...I forgot to remove and move everything in the main function. Nice spotting ;)

**Pull request merged**